server/node_modules/buffer-writer/.travis.yml

```diff
@@ -0,0 +1,7 @@
+language: node_js
```

Comment:
@kvptkr @nicolasabousawan should these node_modules files all be committed here?

Reply:
ran git rm and verified that node_modules is in the .gitignore file